doc/community/user-stats.rst

```diff
@@ -12,3 +12,11 @@ The data we collect is limited to:
 - A generated UUID, randomized when the initial install takes place, no personal or any identifiable information.
 - Environment variables: The OS architecture Ploomber is used in (Python version etc.)
 - Information about the different product phases: installation, API calls and errors.
+
+Version updates
+***************
```

Comment:
this should be level-two header: 

```
something
-----------
```

`***` is level-3 and won't be displayed in the right navigation bar